environments/64-test/public.yml

```diff
@@ -78,7 +78,7 @@ data_email: droberts+datatree@dimagi.com
 subscription_change_email: droberts+accounts+subchange@dimagi.com
 internal_subscription_change_email: droberts+accounts+subchange+internal@dimagi.com
 billing_email: droberts+billing-comm@dimagi.com
-invoicing_contact_email: droberts+billing-support@dimagi.com
+invoicing_contact_email: droberts+accounts@dimagi.com
```

Comment:
FYI @dannyroberts, you may need to update mail filters for this.

Reply:
Thanks. But also, eh, this "environment" is defunct anyway. Can probably just remove it entirely.